apps/wallet/src/stacks/Explorer/index.tsx

```diff
@@ -26,16 +24,6 @@ export const ExplorerStack = () => {
 		unmountOnBlur: false,
 	};
 
-	const ManagedCollectionScreen = withStackContainer(CollectionScreen, {
-		title: 'Collection',
-		goBack: navigateBack,
-	});
-
-	const ManagedCollectibleScreen = withStackContainer(CollectibleScreen, {
-		title: 'Collectible',
-		goBack: navigateBack,
-	});
-
```

Comment:
I see things here, the problem might be caused by `withStackContainer`, please checking it again

Reply:
what problem are you concerned about?

Reply:
i've already removed it